tools/webots.py

```diff
@@ -100,7 +105,11 @@ def exec_build(target, roles, clean):
 
     # Compiles code for correct target
     print("Building code...")
-    exit_code = subprocess.run(["./b", "build"]).returncode
+    build_command = ["./b", "build"]
+    # Check if a -j value has been given
+    if jobs:
+        build_command.extend(["-j", jobs])
```

Comment:
```suggestion
    build_command = ["./b", "build", "-j", jobs]
```

Reply:
We had it like this before, but it seemed more proper to me to just let ninja handle it in the event that we don't supply a flag. What's your thinking behind specifying it every time?

Reply:
The idea here was to keep things clean if we need to add more flags, rather than having to write out the build command each time, just extend each time we add another.
I think @KipHamiltons could explain it a bit better.

Reply:
This suggestion leaves us open to that. It'll have to be done differently if there were flags we wanted to add which didn't have sensible defaults though - they would have to be added separately with an if statement (inline `if`s are not ok, so they don't count). However, I am not sure if that case is relevant/would ever happen.

Reply:
> 
> 
> We had it like this before, but it seemed more proper to me to just let ninja handle it in the event that we don't supply a flag. What's your thinking behind specifying it every time?

Is this comment meant for the other discussion?